client/request_test.go

```
@@ -441,6 +543,39 @@
+func Benchmark_Request_PathParams(b *testing.B) {
+	req := AcquireRequest()
+	req.SetPathParams(map[string]string{
+		"foo": "bar",
+		"bar": "foo",
+	})
+
+	b.ResetTimer()
+	b.ReportAllocs()
+
+	for i := 0; i < b.N; i++ {
+		for range req.PathParams() {
```

golangci-lint (1.62.2)

[warning] empty-block: this block is empty, you can remove it

(revive)